Yoast/javascript
# Improve list of non-passive exceptions for Turkish passive voice

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Added diacritics to the list of full form non-passive exceptions for Turkish, adds more non-passive exception words to the list.

* [Yoast SEO Free] Improves passive voice recognition for Turkish.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Turkish.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`Çanta trende unutuldu.`
`Durumum beğenilmiş.`
`Bu sabah pencere açıldı.`
`Suç hakim tarafından affedilecek.`
`Görüşleri yargıç tarafından dinleniyor.`
`Sorun servis tarafından en kısa sürede çözülebilir.`
`Birçok girişimde bulunuldu.`
`Argümanlar başlatılabilir.`
`Öneri komisyona gönderildi.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add these sentences and check that they do not get highlighted:
`Daha erişilebilir seçenekler mevcuttur.` 
`Onun için o dünyaya bağlanmak çok kolay.` 
`Müzik ve koreografi öğrenmek bir şekilde beynimi açtı.`


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-755
